ts/components/SectionStatusComponent.tsx

```diff
@@ -16,6 +21,11 @@ import { Label } from "./core/typography/Label";
 
 type OwnProps = {
   sectionKey: SectionStatusKey;
+  onSectionRef?: (ref: React.RefObject<View>) => void;
+  navigationProps?: NavigationScreenProp<
```

Comment:
```suggestion
  navigationProps?: NavigationScreenProp<NavigationState>;
```
Could it be a good type too?

Reply:
Super! PR aligned as recommended ✅